pulp_container/app/registry_api.py

```diff
@@ -803,93 +793,157 @@ def put(self, request, path, pk=None):
         if request.content_type not in (
```

Comment:
can you move this check before we receive the artifact and load the json?

Reply:
I am moving it before `raw_data = artifact_file.read()`. We need to know the digest of an artifact before checking the content type beforehand when we want to do this: `raise ManifestInvalid(digest=manifest_digest)`.